pages.it/windows/whoami.md

```diff
@@ -0,0 +1,28 @@
+# whoami
+
+> Mostra informazioni riguardo all'utente utilizzato.
+> Maggiori informazioni: <https://learn.microsoft.com/it-it/windows-server/administration/windows-commands/whoami>.
+
+- Mostra il nome dell'utente utilizzato:
```

Comment:
```suggestion
- Mostra il nome dell'utente corrente:
```

Using "corrente" is better, can you please change all occurrences of "utilizzato" into "corrente"?